frontend/src/LogInSignUp/LogInSignUp.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import './LogInSignUp.css'
+
+import user_icon from '../assets/person.png'
+import email_icon from '../assets/email.png'
+import password_icon from '../assets/password.png'
+
+const LogInSignUp = () =>{
+    const [action,setAction] = useState('Sign Up');
+    return(
+        <div className='body'>
+
+        <div className="container">
+            <div className='header'>
+                <div className="lable">{action}</div>
+                <div className="underline"></div>
+            </div>
+
+            <div className="SignUp-inputs">
+                {action==="Login"?<div></div>:<div className="input">
+                    <img src={user_icon} alt=""/>
+                    <input type="text" placeholder="Name" />
+                </div>
+}
+
+                <div className="input">
+                    <img src={email_icon} alt=""/>
+                    <input type="email" placeholder="Email"/>
+                </div>
+
+                <div className="input">
+                    <img src={password_icon} alt=""/>
+                    <input type="password" placeholder="Password" />
+                </div>
+
+                {action==="Sign Up"?<div></div>:<div className="ForgotPassword">Forgot Password <span>Click Here!</span></div>}
```

Comment:
Would be nice to add some spaces between the signs particularly to make it more readable. In this case it would be `{action === "Sign Up" ? <div></div> : <div...</div>}`
As you see there are spaces in front and in the end of `===` sign, `:`, `?`, etc and it makes code more readable